sklearn/neighbors/regression.py

```diff
@@ -122,8 +122,8 @@ def predict(self, X):
 
         Parameters
         ----------
-        X : array or matrix, shape = [n_samples, n_features]
-
+        X : array of shape [n_samples, n_features]
```

Comment:
It looks like I've pasted the wrong thing here in any case; thanks for almost catching it!